src/components/extensive/Tables/SitesTable.tsx

```diff
@@ -22,22 +22,28 @@ import { ModalId } from "../Modal/ModalConst";
 interface SitesTableProps {
   project: ProjectLightDto;
   hasAddButton?: boolean;
+  onFetch?: (data: EntityIndexConnection<SiteLightDto>) => void;
 }
 
-const SitesTable = ({ project, hasAddButton = true }: SitesTableProps) => {
+const SitesTable = ({ project, hasAddButton = true, onFetch }: SitesTableProps) => {
   const t = useT();
   const { format } = useDate();
-  const [queryParams, setQueryParams] = useState<EntityIndexConnectionProps>({});
+  const [tableParams, setTableParams] = useState<EntityIndexConnectionProps>({});
   const { openModal, closeModal } = useModalContext();
 
-  const [isLoaded, { entities: sites, refetch, indexTotal }] = useSiteIndex({
-    filter: { projectUuid: project.uuid } as any,
-    ...(queryParams as any)
-  });
+  const siteIndexQueryParams = {
+    filter: { projectUuid: project.uuid },
+    ...tableParams
+  };
+  const [isLoaded, response] = useSiteIndex(siteIndexQueryParams as EntityIndexConnectionProps);
```

Comment:
:art: When not destructing a connection, I prefer a more explicit name. "response" isn't really accurate either: what's coming from the connection is cached data in the store, and often doesn't really match up directly with what came from the API, nor does the component care what the direct API response looks like. Something like `siteIndexConnection` or `siteIndex` would be more appropriate imo.

Reply:
renamed